### PR TITLE
Add edit button

### PR DESCRIPTION
**Pull request by JosephToscano1**
I added an edit button that would appear for each reflection and allow the user to edit the textbox for the corresponding reflection, and the new value would be saved to LocalStorage.

Instructions for PR Team:
1. Open with Live Server and navigate to Reflect on the Day
2. Create a reflection
3. Verify that an 'edit' button appears next to the delete button for a reflection
4. Enter some text
5. Refresh the page
6. Use the edit button to enable a reflection's textbox
7. Edit reflection content, click off box, then refresh
8. Verify that new content has been saved

Notes: The edit button does not allow the user to replace the sentiment icon for a reflection. Unsure whether or not that is necessary

This branch can be deleted upon a successful merge.

### CODE REVIEW

**Review by yvcardenas — COMMENTED** [review]
Do you want me to accept this now and you guys add the edit widget button later? Or do you prefer to wait until both are implemented to approve? Maybe when you press the widget all the options pop up again.

**Comment by JosephToscano1** [comment]
I think accepting now and adding the widget button later works!

**Review by yvcardenas — APPROVED** [review]

**Pull request merged**
